src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 auto;
```

Comment:
redundant

Reply:
without display flex and space between will not work correctly